src/main/java/de/helfenkannjeder/come2help/server/domain/Ability.java

```diff
@@ -0,0 +1,89 @@
+package de.helfenkannjeder.come2help.server.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+public class Ability extends AbstractVersionedAuditable {
+
+    @Id
+    @GeneratedValue
+    private Long id = null;
+
+    @NotNull
+    private String name;
+
+    private String description;
+
+    @ManyToMany(mappedBy="abilities")
+    private List<User> users;
```

Comment:
Is this relation on this position necessary? Typically you need this only with additional restrictions (location etc.)